### DISCUSSION TITLE
body force

### DISCUSSION THREAD

**Discussion by rushouzhichen** [peridigm/peridigm#382] [Q&A]
Hello, while studying Peridigm, I attempted to calculate the benchmark problem in Chapter 8 of *Peridynamics: Theory and Its Application*. However, I found that no matter how I modified the XML input file, the results never matched those in the book. Taking the example of a tensioned rod, here is the input file I wrote and the corresponding results.
[1d.txt](https://github.com/user-attachments/files/18051040/1d.txt)
![image](https://github.com/user-attachments/assets/8970433b-a605-4afe-87a7-2f962eb22139)
The body force is applied to a single material point at the right end. Based on the applied body force and the volume of the material point, it matches the 200 N applied in the benchmark example from the book. However, I still cannot figure out what the issue is. Similarly, for an isotropic plate under uniaxial tension, the results also do not match those in the book's examples.

**Comment by franciscoyapor** [comment]
Addressed in [Issue #383](https://github.com/peridigm/peridigm/issues/383#issuecomment-2543214924)
